bundle/manifests/numaresources-operator.clusterserviceversion.yaml

```diff
@@ -615,11 +615,7 @@ spec:
           resources:
           - networkpolicies
           verbs:
-          - create
-          - delete
-          - get
-          - list
-          - update
+          - '*'
```

Comment:
it is still better to list all the verbs explicitly, but I don't mind enough now. Let's move on.